### PR TITLE
Update README to include goals of this Workgroup

### PR DESCRIPTION
**Pull request by p0nce**
Hello,

I have updated README to include goals.

People have been confused with what this WG is about.

Some of that is a kind of manifesto that maybe is a personal interpretation. **Feel free to disagree here and ask for mods here.**
I feel there will be a need to classify what is important formally, and in which order (for example 1. performance 2. unopinionated 3. feeling nice about ourselves). Languages usually list those reasons.

**Please vote with upvote/downvote this PR.**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,50 @@
...
+
+We feel that Phobos leads the path towards building reusable, expressive abstractions;
+and that there are no reason a GUI Toolkit couldn't be _unopinionated_ to the same degree
+ std.range or std.allocator aren't.
+
+We acknowledge that working on our own graphics libraries (like we currently do), may not yield 
+the best possible result for everyone; and that discussion and arguing are called 
+for if we are to keep the D community engaged.
+
+Because this task is essentially about defining interfaces, and those interfaces exchanges "images",
```
Comment (igor84) [inline README.md]: ... about defining *graphical* interfaces ...

File: README.md [hunk 2]
```diff
@@ -1,11 +1,50 @@
...
+
+# Goals
+
+
+The Dlang Graphics Workgroup is a collective of D library writers who want to
+make libraries together with a collegial design.
+
+We feel that Phobos leads the path towards building reusable, expressive abstractions;
+and that there are no reason a GUI Toolkit couldn't be _unopinionated_ to the same degree
+ std.range or std.allocator aren't.
```
Comment (ahmetsait) [inline README.md]: Better wording:
there is no reason a GUI Toolkit can't be unopinionated to the same degree std.range or std.allocator is.

File: README.md (hunk 1 above)
Comment (p0nce) [reply README.md]: fixed, thanks

File: README.md (hunk 2 above)
Comment (p0nce) [reply README.md]: Thanks

**Comment by p0nce** [comment]
3 votes for, 0 votes against.

**Pull request merged**